### DISCUSSION TITLE
How many curStep(s) happen per second or how to calculate seconds with curStep?

### DISCUSSION THREAD

**Discussion by DestDeHat** [KadeArchive/Kade-Engine#1233] [Q&A]
How many curSteps happen every second/How to calculate seconds with the curSteps? I cannot find a answer anywhere else so i figure i'd post here.

**Comment by isakube** [comment]
Conductor has vars named `crochet` and `stepCrochet` which measure the time inbetween beats and steps im milliseconds, respectively. stepCrochet / 1000 gets you how many seconds between each step
